docs/content/utilities/batch.mdx

```diff
@@ -0,0 +1,95 @@
+---
+title: Batch
+description: Utility
+---
+
+import Note from "../../src/components/Note"
+
+The batch utility provides an abstraction to process a batch event. Useful for lambda integrations with [AWS SQS](https://aws.amazon.com/sqs/), [AWS Kinesis](https://aws.amazon.com/kinesis/) and [AWS DynamoDB Streams](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/Streams.html).
+It also provides base classes (`BaseProcessor`, `BasePartialProcessor`) allowing you to create your **own** batch processor.
+
+**Key Features**
+
+* Run batch processing logic with a clean interface;
```

Comment:
Expand this - I think what you meant is:

"Bring your own batch processor with <some included features/ctx manager/failure/success etc>"